linode/data_source_linode_images_test.go

```diff
@@ -0,0 +1,100 @@
+package linode
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"fmt"
+	"strconv"
+	"testing"
+)
+
+func TestAccDataSourceLinodeImages_basic(t *testing.T) {
+	t.Parallel()
+
+	imageID := "linode/debian8"
+	resourceName := "data.linode_images.foobar"
+
+	resource.Test(t, resource.TestCase{
```

Comment:
This test could fail with later changes. It's probably best to build an image of our own and test filtering for it. See `resource_linode_image_test.go`